packages/pigeon/platform_tests/alternate_language_test_plugin/ios/Classes/AlternateLanguageTestPlugin.m

```diff
@@ -132,6 +132,16 @@ - (void)echoAsyncString:(NSString *)aString
   completion(aString, nil);
 }
 
+- (void)throwAsyncErrorWithCompletion:(void (^)(id _Nullable, FlutterError *_Nullable))completion {
+  FlutterError *_Nullable error = [FlutterError errorWithCode:@"An error" message:nil details:nil];
+  completion(nil, error);
```

Comment:
Optional nit: you can inline this: `completion(nil, [FlutterError errorWithCode:@"An error" message:nil details:nil]);`

(Same below.)